cmd/search/main.go

```diff
@@ -284,12 +284,27 @@ func fetchJob(client *http.Client, job *ProwJob, indexedPaths *pathIndex, toDir
 		return nil
 	}
 
+	pathOnDisk := filepath.Join(append([]string{toDir}, strings.Split(logPath, "/")...)...)
```

Comment:
This is a bug in something with indexedPaths.  IndexedPaths.MetadataFor should return true if the file exists, so if we're getting here that means there's a mismatch between the path we write and the path we index.  The path we index should be `/origin-ci-test/logs/....`

Reply:
> ...there's a mismatch between the path we write and the path we index...

Maybe #17 fixes this, then?

Reply:
/hold

while we sort out wether #17 is good enough.